contrib/docker/tester/build.sh

```diff
@@ -1,8 +1,5 @@
 #!/bin/bash
 
 # This script generates a docker image for the ASPECT tester.
-# It requires a docker installation on the local machine, and the ability to
-# communicate with the docker daemon without root user privileges (see the docker
-# webpage for an explanation).
 
-docker build -t geodynamics/aspect-tester:focal-dealii-9.4-v2 .
+docker build -t geodynamics/aspect-tester:latest .
```

Comment:
why is this ``latest`` and not ``focal-dealii-master``?

Reply:
My bad I did not check that file. Fixed now.